### PR TITLE
HDXDSYS-1384 Update HAPI poverty rate docs with info about trends falling back on mpi data

### PR DESCRIPTION
**Pull request by mcarans**
Poverty rate documentation update to reflect that poverty rate data is taken from trends data falling back on latest data and that it is at admin 0 and 1

Also food security documentation updated to describe how p-coding works

### CODE REVIEW

File: docs/data_usage_guides/food_security_and_nutrition.md [hunk 1]
```diff
@@ -31,10 +31,15 @@ For available query parameters, please see the
...
   which uses phonetic name matching and manual overrides
-* Any rows whose p-codes are unmatched are p-coded at the national level.
+* Where admin 1 names could not be p-coded, the provided p-codes from the 
+  source data are at national level
+* Where admin 2 names could not be p-coded, the provided p-codes from the 
+  source data are at admin 1 level if possible or national level if not
 
 ### Usage Notes
 
+* The data is available at national, admin 1 and admin 2 with admin names
```
Comment (turnerm) [inline docs/data_usage_guides/food_security_and_nutrition.md:41]: should be "admin 2 _levels_"

File: docs/data_usage_guides/food_security_and_nutrition.md [hunk 2]
```diff
@@ -31,10 +31,15 @@ For available query parameters, please see the
...
-* Any rows whose p-codes are unmatched are p-coded at the national level.
+* Where admin 1 names could not be p-coded, the provided p-codes from the 
+  source data are at national level
+* Where admin 2 names could not be p-coded, the provided p-codes from the 
+  source data are at admin 1 level if possible or national level if not
 
 ### Usage Notes
 
+* The data is available at national, admin 1 and admin 2 with admin names 
+  supplied in the returned data along with p-codes where available
```
Comment (turnerm) [inline docs/data_usage_guides/food_security_and_nutrition.md:42]: which admin names supplied do you mean - should we say, both provider and standardized?

File: docs/data_usage_guides/population_and_socio-economy.md [hunk 3]
```diff
@@ -73,6 +75,14 @@ For available query parameters, please see the
 
 ### Usage Notes
 
-* The data is disaggregated to admin 1, but not p-coded. We have kept the
-  admin 1 names in the data, but link only to national level p-codes.
-  We plan to p-code this data in a future release.
+* The data is available at admin 0 and admin 1 with admin names supplied in the
```
Comment (turnerm) [inline docs/data_usage_guides/population_and_socio-economy.md:78]: "at the country and admin 1 level"

**Review by turnerm — APPROVED** [review]
Thanks so much for taking care of this Mike.

I believe the poverty rate parameters returned table also needs to be updated to reflect the additional columns now that we're p-coding.

**Review by alexandru-m-g — APPROVED** [review]

**Pull request merged**